libs/math/tests/test_half.cpp

```diff
@@ -90,3 +90,25 @@ TEST_F(HalfTest, Vec) {
     EXPECT_EQ(f4.xyz, h3);
     EXPECT_EQ(f4.xy, h2);
 }
+
+
+using fp10 = fp<0, 5, 5>;
```

Comment:
Should we declared those in `half.h`?

Reply:
no because fp<> is just a utility, it doesn't provide the full std::numeric_limits<> support